ru/turing/courses/lesson2/animals/Animal.java

```diff
@@ -1,4 +1,72 @@
 package ru.turing.courses.lesson2.animals;
 
 public class Animal {
+    protected String klichka;
```

Comment:
```suggestion
    protected String nickname;
```
Называть по возможности словами на английском языке!

Reply:
!